pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -174,7 +174,7 @@ def __init__(
             "min": (torch_inf, "min"),
             "max": (-torch_inf, "max"),
             "auto": (-torch_inf, "max")
-            if "acc" in self.monitor or self.monitor.startswith("fmeasure")
+            if monitor is not None and ("acc" in monitor or monitor.startswith("fmeasure"))
```

Comment:
what is the default behaviour for `None`?

Reply:
None means no quantity is monitored so `mode` has no use.